util/winutil/winutil_windows.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build windows
```

Comment:
this is fine but these lines are unnecessary when you the filename is already a _GOOS.go or _GOARCH.go or _GOOS_GOARCH.go suffix as it is in this case.

see https://pkg.go.dev/cmd/go#hdr-Build_constraints